src/mantarray_desktop_app/utils/generic.py

```diff
@@ -122,7 +122,7 @@ def validate_user_credentials(request_args: Dict[str, Any]) -> None:
     if customer_id := request_args.get("customer_id"):
         user_name = request_args["user_name"]
         user_password = request_args["user_password"]
-        get_cloud_api_tokens(customer_id, user_name, user_password)
+        return get_cloud_api_tokens(customer_id, user_name, user_password)
```

Comment:
I think we should also add a `return None` somewhere in here so it's a little more clear what's going on